### ISSUE TITLE
PANIC when calling X/addresses

### ISSUE THREAD

**Issue by tbrunain** [ava-labs/ortelius#19]
Hello,

I finally managed to start Ortelius locally, but when I call /X/addresses it crash with the following error : 

```
api_1              | INFO [07-10|16:17:06] /go/src/github.com/ava-labs/ortelius/api/server.go#59: Server listening on :8080
stream-consumer_1  | [2020-07-10T16:17:06.489773002Z]: job:bootstrap event:dbr.exec time:1427 μs kvs:[chain_id:rrEWX7gc7D9mwcdrdBxBTdqh1a7WDVsMuadhTZgyXfFcRz45L sql:INSERT INTO `avm_outputs` (`id`,`chain_id`,`transaction_id`,`output_index`,`asset_id`,`output_type`,`amount`,`created_at`,`locktime`,`threshold`) VALUES ('yDzUcttbbkkedkFfF3SewV7SM1kKhNaH52Hfkqkmw15LKSuSK','rrEWX7gc7D9mwcdrdBxBTdqh1a7WDVsMuadhTZgyXfFcRz45L','21d7KVtPrubc5fHr6CGNcgbUb4seUjmZKr35ZX7BZb5iP8pXWA',9,'21d7KVtPrubc5fHr6CGNcgbUb4seUjmZKr35ZX7BZb5iP8pXWA',255,45000000000000000,'2020-07-10 16:17:06.000000',0,1)]
stream-consumer_1  | [2020-07-10T16:17:06.490852831Z]: job:bootstrap event:dbr.exec time:1003 μs kvs:[chain_id:rrEWX7gc7D9mwcdrdBxBTdqh1a7WDVsMuadhTZgyXfFcRz45L sql:INSERT INTO `avm_output_addresses` (`output_id`,`address`) VALUES ('yDzUcttbbkkedkFfF3SewV7SM1kKhNaH52Hfkqkmw15LKSuSK','DpL8PTsrjtLzv5J8LL3D2A6YcnCTqrNH9')]
stream-consumer_1  | [2020-07-10T16:17:06.492047253Z]: job:bootstrap event:dbr.exec time:1124 μs kvs:[chain_id:rrEWX7gc7D9mwcdrdBxBTdqh1a7WDVsMuadhTZgyXfFcRz45L sql:INSERT INTO `avm_outputs` (`id`,`chain_id`,`transaction_id`,`output_index`,`asset_id`,`output_type`,`amount`,`created_at`,`locktime`,`threshold`) VALUES ('Rb3ysWb2GcsHXGU6wePDZwKU6y1hEhRsQzjfyZ9EFsYENHtKf','rrEWX7gc7D9mwcdrdBxBTdqh1a7WDVsMuadhTZgyXfFcRz45L','21d7KVtPrubc5fHr6CGNcgbUb4seUjmZKr35ZX7BZb5iP8pXWA',10,'21d7KVtPrubc5fHr6CGNcgbUb4seUjmZKr35ZX7BZb5iP8pXWA',255,45000000000000000,'2020-07-10 16:17:06.000000',0,1)]
stream-consumer_1  | [2020-07-10T16:17:06.498880712Z]: job:bootstrap event:dbr.exec time:6 ms kvs:[chain_id:rrEWX7gc7D9mwcdrdBxBTdqh1a7WDVsMuadhTZgyXfFcRz45L sql:INSERT INTO `avm_output_addresses` (`output_id`,`address`) VALUES ('Rb3ysWb2GcsHXGU6wePDZwKU6y1hEhRsQzjfyZ9EFsYENHtKf','JLrYNMYXANGj43BfWXBxMMAEenUBp1Sbn')]
stream-consumer_1  | [2020-07-10T16:17:06.519417243Z]: job:bootstrap event:dbr.exec time:20 ms kvs:[chain_id:rrEWX7gc7D9mwcdrdBxBTdqh1a7WDVsMuadhTZgyXfFcRz45L sql:INSERT INTO `avm_assets` (`id`,`chain_Id`,`name`,`symbol`,`denomination`,`alias`,`current_supply`) VALUES ('21d7KVtPrubc5fHr6CGNcgbUb4seUjmZKr35ZX7BZb5iP8pXWA','rrEWX7gc7D9mwcdrdBxBTdqh1a7WDVsMuadhTZgyXfFcRz45L','AVA','AVA',9,'AVA',450000000000000000)]
stream-consumer_1  | [2020-07-10T16:17:06.525772515Z]: job:bootstrap event:dbr.exec time:6 ms kvs:[chain_id:rrEWX7gc7D9mwcdrdBxBTdqh1a7WDVsMuadhTZgyXfFcRz45L sql:INSERT INTO `avm_transactions` (`id`,`chain_id`,`type`,`created_at`,`canonical_serialization`) VALUES ('21d7KVtPrubc5fHr6CGNcgbUb4seUjmZKr35ZX7BZb5iP8pXWA','rrEWX7gc7D9mwcdrdBxBTdqh1a7WDVsMuadhTZgyXfFcRz45L','create_asset','2020-07-10 16:17:06.000000',?)]
stream-consumer_1  | [2020-07-10T16:17:06.545563423Z]: job:bootstrap event:dbr.commit kvs:[chain_id:rrEWX7gc7D9mwcdrdBxBTdqh1a7WDVsMuadhTZgyXfFcRz45L]
stream-consumer_1  | [2020-07-10T16:17:06.545619909Z]: job:bootstrap status:success time:128 ms kvs:[chain_id:rrEWX7gc7D9mwcdrdBxBTdqh1a7WDVsMuadhTZgyXfFcRz45L]
stream-consumer_1  | ERROR[07-10|16:17:16] /go/src/github.com/ava-labs/ortelius/stream/processor.go#153: Unknown error: context deadline exceeded
stream-consumer_1  | ERROR[07-10|16:17:26] /go/src/github.com/ava-labs/ortelius/stream/processor.go#153: Unknown error: context deadline exceeded
stream-consumer_1  | ERROR[07-10|16:17:36] /go/src/github.com/ava-labs/ortelius/stream/processor.go#153: Unknown error: context deadline exceeded
stream-consumer_1  | ERROR[07-10|16:17:46] /go/src/github.com/ava-labs/ortelius/stream/processor.go#153: Unknown error: context deadline exceeded
stream-consumer_1  | ERROR[07-10|16:17:56] /go/src/github.com/ava-labs/ortelius/stream/processor.go#153: Unknown error: context deadline exceeded
stream-consumer_1  | ERROR[07-10|16:18:06] /go/src/github.com/ava-labs/ortelius/stream/processor.go#153: Unknown error: context deadline exceeded
stream-consumer_1  | ERROR[07-10|16:18:16] /go/src/github.com/ava-labs/ortelius/stream/processor.go#153: Unknown error: context deadline exceeded
stream-consumer_1  | ERROR[07-10|16:18:26] /go/src/github.com/ava-labs/ortelius/stream/processor.go#153: Unknown error: context deadline exceeded
stream-consumer_1  | ERROR[07-10|16:18:36] /go/src/github.com/ava-labs/ortelius/stream/processor.go#153: Unknown error: context deadline exceeded
stream-consumer_1  | ERROR[07-10|16:18:46] /go/src/github.com/ava-labs/ortelius/stream/processor.go#153: Unknown error: context deadline exceeded
stream-consumer_1  | ERROR[07-10|16:18:56] /go/src/github.com/ava-labs/ortelius/stream/processor.go#153: Unknown error: context deadline exceeded
stream-consumer_1  | ERROR[07-10|16:19:06] /go/src/github.com/ava-labs/ortelius/stream/processor.go#153: Unknown error: context deadline exceeded
stream-consumer_1  | ERROR[07-10|16:19:16] /go/src/github.com/ava-labs/ortelius/stream/processor.go#153: Unknown error: context deadline exceeded
stream-consumer_1  | ERROR[07-10|16:19:26] /go/src/github.com/ava-labs/ortelius/stream/processor.go#153: Unknown error: context deadline exceeded
stream-consumer_1  | ERROR[07-10|16:19:36] /go/src/github.com/ava-labs/ortelius/stream/processor.go#153: Unknown error: context deadline exceeded
stream-consumer_1  | ERROR[07-10|16:19:46] /go/src/github.com/ava-labs/ortelius/stream/processor.go#153: Unknown error: context deadline exceeded
stream-consumer_1  | ERROR[07-10|16:19:56] /go/src/github.com/ava-labs/ortelius/stream/processor.go#153: Unknown error: context deadline exceeded
api_1              | [2020-07-10T16:19:59.331351046Z]: job:request.x.addresses status:success time:554 μs
api_1              | [2020-07-10T16:19:59.662912197Z]: job:request.favicon.ico status:success time:38 μs
api_1              | [2020-07-10T16:20:05.459470988Z]: job:request. status:success time:22 μs
stream-consumer_1  | ERROR[07-10|16:20:06] /go/src/github.com/ava-labs/ortelius/stream/processor.go#153: Unknown error: context deadline exceeded
api_1              | [2020-07-10T16:20:09.872044923Z]: job:request.X. status:success time:147 μs
stream-consumer_1  | ERROR[07-10|16:20:16] /go/src/github.com/ava-labs/ortelius/stream/processor.go#153: Unknown error: context deadline exceeded
api_1              | ERROR 2020/07/10 16:20:16.710169 panic_handler.go:26: PANIC
api_1              | URL: /X/addresses
api_1              | ERROR: runtime error: invalid memory address or nil pointer dereference
api_1              | STACK:
api_1              | goroutine 11 [running]:
api_1              | github.com/gocraft/web.(*Router).handlePanic(0xc0004ffba0, 0xc000198180, 0xc0001981a0, 0xce3100, 0x16c1110)
api_1              | 	/go/pkg/mod/github.com/gocraft/web@v0.0.0-20190207150652-9707327fb69b/router_serve.go:277 +0x4fc
api_1              | github.com/gocraft/web.(*Router).ServeHTTP.func1(0xc0004ffba0, 0xc000198180)
api_1              | 	/go/pkg/mod/github.com/gocraft/web@v0.0.0-20190207150652-9707327fb69b/router_serve.go:43 +0x6f
api_1              | panic(0xce3100, 0x16c1110)
api_1              | 	/usr/local/go/src/runtime/panic.go:969 +0x166
api_1              | github.com/ava-labs/ortelius/services/cache.(*Cache).Get(0x0, 0x1059320, 0xc0002f81e0, 0xc00027c180, 0x59, 0x59, 0xc00001e000, 0xd06840, 0xd71420, 0x1055160)
api_1              | 	/go/src/github.com/ava-labs/ortelius/services/cache/cache.go:46 +0x4a
api_1              | github.com/ava-labs/ortelius/api.(*RootRequestContext).WriteCacheable(0xc000186700, 0x1055160, 0xc000198180, 0xc0002f83c0, 0x5, 0x6, 0xc0001924f8)
api_1              | 	/go/src/github.com/ava-labs/ortelius/api/root.go:55 +0xb5
api_1              | github.com/ava-labs/ortelius/services/indexes/avm.(*APIContext).ListAddresses(0xc000072820, 0x10624c0, 0xc000198180, 0xc0001981a0)
api_1              | 	/go/src/github.com/ava-labs/ortelius/services/indexes/avm/api.go:169 +0x263
api_1              | reflect.Value.call(0xcb8660, 0xf4b950, 0x13, 0xded953, 0x4, 0xc000192980, 0x3, 0x3, 0x45a9c7, 0xcda140, ...)
api_1              | 	/usr/local/go/src/reflect/value.go:460 +0x8ab
api_1              | reflect.Value.Call(0xcb8660, 0xf4b950, 0x13, 0xc000192980, 0x3, 0x3, 0x101000007dc20, 0xc88c20, 0xc000192990)
api_1              | 	/usr/local/go/src/reflect/value.go:321 +0xb4
api_1              | github.com/gocraft/web.middlewareStack.func1(0x10624c0, 0xc000198180, 0xc0001981a0)
api_1              | 	/go/pkg/mod/github.com/gocraft/web@v0.0.0-20190207150652-9707327fb69b/router_serve.go:139 +0x3a1
api_1              | github.com/ava-labs/ortelius/services/indexes/avm.NewAPIRouter.func1(0xc000072820, 0x10624c0, 0xc000198180, 0xc0001981a0, 0xc0001b22f0)
api_1              | 	/go/src/github.com/ava-labs/ortelius/services/indexes/avm/api.go:43 +0xa7
api_1              | reflect.Value.call(0xcce860, 0xc0000bd380, 0x13, 0xded953, 0x4, 0xc000192d60, 0x4, 0x4, 0xc000388205, 0xd3c500, ...)
api_1              | 	/usr/local/go/src/reflect/value.go:460 +0x8ab
api_1              | reflect.Value.Call(0xcce860, 0xc0000bd380, 0x13, 0xc000192d60, 0x4, 0x4, 0xc000186700, 0x16, 0xc000192dc0)
api_1              | 	/usr/local/go/src/reflect/value.go:321 +0xb4
api_1              | github.com/gocraft/web.(*middlewareHandler).invoke(0xc0003d33e0, 0xdac240, 0xc000072820, 0x16, 0x10624c0, 0xc000198180, 0xc0001981a0, 0xc0001b22f0)
api_1              | 	/go/pkg/mod/github.com/gocraft/web@v0.0.0-20190207150652-9707327fb69b/router_serve.go:159 +0x26a
api_1              | github.com/gocraft/web.middlewareStack.func1(0x10624c0, 0xc000198180, 0xc0001981a0)
api_1              | 	/go/pkg/mod/github.com/gocraft/web@v0.0.0-20190207150652-9707327fb69b/router_serve.go:148 +0x18b
api_1              | github.com/ava-labs/ortelius/api.(*RootRequestContext).setHeaders(0xc000186700, 0x10624c0, 0xc000198180, 0xc0001981a0, 0xc0001b22f0)
api_1              | 	/go/src/github.com/ava-labs/ortelius/api/root.go:158 +0x26d
api_1              | reflect.Value.call(0xcce800, 0xf4b920, 0x13, 0xded953, 0x4, 0xc000193350, 0x4, 0x4, 0x0, 0x3, ...)
api_1              | 	/usr/local/go/src/reflect/value.go:460 +0x8ab
api_1              | reflect.Value.Call(0xcce800, 0xf4b920, 0x13, 0xc000193350, 0x4, 0x4, 0x7097432def9a94c6, 0x300000002, 0x203000)
api_1              | 	/usr/local/go/src/reflect/value.go:321 +0xb4
api_1              | github.com/gocraft/web.(*middlewareHandler).invoke(0xc0002f7ad0, 0xd3c500, 0xc000186700, 0x16, 0x10624c0, 0xc000198180, 0xc0001981a0, 0xc0001b22f0)
api_1              | 	/go/pkg/mod/github.com/gocraft/web@v0.0.0-20190207150652-9707327fb69b/router_serve.go:159 +0x26a
api_1              | github.com/gocraft/web.middlewareStack.func1(0x10624c0, 0xc000198180, 0xc0001981a0)
api_1              | 	/go/pkg/mod/github.com/gocraft/web@v0.0.0-20190207150652-9707327fb69b/router_serve.go:148 +0x18b
api_1              | github.com/ava-labs/ortelius/api.newContextSetter.func1(0xc000186700, 0x10624c0, 0xc000198180, 0xc0001981a0, 0xc0001b22f0)
api_1              | 	/go/src/github.com/ava-labs/ortelius/api/root.go:115 +0x1b3
api_1              | reflect.Value.call(0xcce800, 0xc000188f80, 0x13, 0xded953, 0x4, 0xc000193958, 0x4, 0x4, 0x412f0b, 0xc000193920, ...)
api_1              | 	/usr/local/go/src/reflect/value.go:460 +0x8ab
api_1              | reflect.Value.Call(0xcce800, 0xc000188f80, 0x13, 0xc000193958, 0x4, 0x4, 0x0, 0x0, 0xc0001939a8)
api_1              | 	/usr/local/go/src/reflect/value.go:321 +0xb4
api_1              | github.com/gocraft/web.(*middlewareHandler).invoke(0xc0002f7aa0, 0xd3c500, 0xc000186700, 0x16, 0x10624c0, 0xc
```

**Comment by DreamHackchosenone** [comment]
@tbrunain can you tell me what is the full url? thanks

**Comment by tyler-smith** [comment]
Thanks for reporting this. I investigate this as soon as I can.

**Comment by tyler-smith** [comment]
Fixed in #53.
